newbank/server/Customer.java

```diff
@@ -24,6 +24,16 @@ public void printTransferableFromAccounts(double requestAmount) {
 		MenuPrinter.printTransferableFrom(transferableFromAccounts);
 	}
 
+	public void printDepositToAccounts(double requestAmount) {
+		ArrayList<Account> depositToAccounts = new ArrayList<Account>();
+		for(Account a : accounts) {
+			if (a.getCurrentBalance() >= requestAmount){
+				depositToAccounts.add(a);
+			}
+		}
+		MenuPrinter.printDepositTo(depositToAccounts);
+	}
```

Comment:
not sure if I'm reading this wrong but don't we want to check `getCurrentBalance() >= requestAmount` only on withdraw, not deposit?

Reply:
Whoops, that's what I get for copying and pasting without looking properly. Removing requestAmount :)

Reply:
nice, in that case we can also remove `ArrayList<Account> depositToAccounts = new ArrayList<Account>();` and just call `MenuPrinter.printDepositTo(accounts);`

Reply:
Thank you, done and tested :)